diff --git a/CMakeLists.txt b/CMakeLists.txt
index 2ff36ab..15a6493 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -132,8 +132,7 @@ FetchContent_Declare(implot        URL https://github.com/epezent/implot/archive
 
 # Server dependencies
 FetchContent_Declare(monado
-    GIT_REPOSITORY   https://gitlab.freedesktop.org/monado/monado
-    GIT_TAG          57e937383967c7e7b38b5de71297c8f537a2489d
+    SOURCE_DIR @monadoSrc@
     )
 
 FetchContent_Declare(imgui
